charts/kong/values.yaml

```diff
@@ -934,19 +951,29 @@ podDisruptionBudget:
 podSecurityPolicy:
```

Comment:
Given that `PodSecurityPolicy` [is removed as of k8s 1.25](https://kubernetes.io/docs/concepts/security/pod-security-policy/) I'd say that I don't see the point of changing these values at all.

Do you mind sharing why would you see this being changed?

Reply:
Back in April 2024, when I created the PR, I was running kong on a 1.24 cluster. I agree this isn't relevant any more, so I removed the changes